src/lib/mina/graphql.ts

```diff
@@ -0,0 +1,492 @@
+import { ActionStatesStringified, removeJsonQuotes } from '../fetch.js';
```

Comment:
This file was meant to hold all types of queries related to GraphQL requests. I thought `fetch.ts` was starting to get bloated and hard to read, so moving some of the static definitions to a separate module would help.